packages/index.bzl

```diff
@@ -0,0 +1,53 @@
+load("@npm_bazel_rollup//:index.bzl", "rollup_bundle")
+load("@build_bazel_rules_nodejs//:index.bzl", "pkg_npm")
+
+def pkg_json(name, package_name, description, version, external_deps = [], layers = [], main = "", types = ""):
+    native.genrule(
+        name = name,
+        srcs = layers,
+        tools = ["//packages:gen-package-json-bin"],
+        outs = ["package.json"],
+        cmd = """$(location //packages:gen-package-json-bin) \\
```

Comment:
check out https://github.com/bazelbuild/rules_nodejs/blob/master/packages/typescript/checked_in_ts_project.bzl#L23-L39 - maybe this is simpler for you